js/app/src/routes/[...catchall]/+page.svelte

```diff
@@ -292,6 +292,16 @@
 			window.parent.postMessage(supports_zerogpu_headers, origin);
 		}
 
+		if (config.js) {
```

Comment:
We also need to add this to `spa/Index.svelte` in order for it to work in spa mode

Reply:
fixed!